docs/usage.md

```diff
@@ -67,88 +84,142 @@ results         # Finished results (configurable, see below)
-    * Pulls most software from [Bioconda](https://bioconda.github.io/)
-* `binac`
-    * A profile for the [BinAC](https://www.bwhpc-c5.de/wiki/index.php/Category:BwForCluster_BinAC) cluster
-    * Pulls images via Singularity from Dockerhub automatically
-* `cfc`
-    * A profile for the Core Facility Cluster at QBiC Tuebingen
-    * Pulls images via Singularity from Dockerhub automatically
-* `awsbatch`
-    * A generic configuration profile to be used with AWS Batch.
+  * Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker or Singularity.
+  * A generic configuration profile to be used with [Conda](https://conda.io/docs/)
+  * Pulls most software from [Bioconda](https://bioconda.github.io/)
 * `test`
-    * A profile with a complete configuration for automated testing
-    * Includes links to test data so needs no other parameters
-* `none`
-    * No configuration at all. Useful if you want to build your own config from scratch and want to avoid loading in the default `base` config profile (not recommended).
+  * A profile with a complete configuration for automated testing
+  * Includes links to test data so needs no other parameters
+
+### `--input`
 
-<!-- TODO nf-core: Document required command line parameters -->
-### `--reads`
-Use this to specify the location of your input FastQ files. For example:
+You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a tab-separated file with minimum 4 columns, and a header row as shown in the examples below.
 
 ```bash
---reads 'path/to/data/sample_*_{1,2}.fastq'
+--input '[path to design file]'
 ```
 
-Please note the following requirements:
+#### Required columns
 
-1. The path must be enclosed in quotes
-2. The path must have at least one `*` wildcard character
-3. When using the pipeline with paired end data, the path must use `{1,2}` notation to specify read pairs.
+The `group` identifier demarks a given celltype or patient in which an experiment was performed and should be identical for all conditions and replicates in which an experiment was performed. The `condition` identifier demarks a given condition such as a control / drug treatment or WT / KO genotype and should be identical for all replicates of the given condition. The `control` identifier is used to demark all replicates of a control `condition` with a given `group` and should be `1` for control replicates and `0` for all others. This is used to set up the proper contrasts for the [DESeq2](https://doi.org/10.1186/s13059-014-0550-8) analysis. The last required column is `reads` pointing to the associated raw read sets in fastq format.
+
+In the design below there a triplicate samples for two groups (`K562` and `OCIAML3`) with two conditions each (one control condition `DMSO` and two treatment conditions `NVP.lo` and `NVP.hi`).
+
```

Comment:
If you want to get fancy, you can do this as a Markdown table and then have the raw TSV in a details block that is hidden by default but can be expanded:

```

| group   | condition | control | reads |
|---------|-----------|---------|-------|
| MOLM-13 | DMSO   | 1 | MOLM-13_dmso_1.fq.gz   |
| MOLM-13 | DMSO   | 1 | MOLM-13_dmso_2.fq.gz   |
| MOLM-13 | DMSO   | 1 | MOLM-13_dmso_3.fq.gz   |
| MOLM-13 | NVP_hi | 0 | MOLM-13_nvp.hi_1.fq.gz |
| MOLM-13 | NVP_hi | 0 | MOLM-13_nvp.hi_2.fq.gz |
| MOLM-13 | NVP_hi | 0 | MOLM-13_nvp.hi_3.fq.gz |
| MOLM-13 | NVP_lo | 0 | MOLM-13_nvp.lo_1.fq.gz |
| MOLM-13 | NVP_lo | 0 | MOLM-13_nvp.lo_2.fq.gz |
| MOLM-13 | NVP_lo | 0 | MOLM-13_nvp.lo_3.fq.gz |
| OCIAML3 | DMSO   | 1 | OCIAML3_dmso_1.fq.gz   |
| OCIAML3 | DMSO   | 1 | OCIAML3_dmso_2.fq.gz   |
| OCIAML3 | DMSO   | 1 | OCIAML3_dmso_3.fq.gz   |
| OCIAML3 | NVP_hi | 0 | OCIAML3_nvp.hi_1.fq.gz |
| OCIAML3 | NVP_hi | 0 | OCIAML3_nvp.hi_2.fq.gz |
| OCIAML3 | NVP_hi | 0 | OCIAML3_nvp.hi_3.fq.gz |
| OCIAML3 | NVP_lo | 0 | OCIAML3_nvp.lo_1.fq.gz |
| OCIAML3 | NVP_lo | 0 | OCIAML3_nvp.lo_2.fq.gz |
| OCIAML3 | NVP_lo | 0 | OCIAML3_nvp.lo_3.fq.gz |

<details>
<summary>Raw TSV:</summary>

-- existing bash block from below (trying to enter this in a comment confused the markdown parsing) --

</details>
```

Reply:
I don't exactly get what you mean here - can you elaborate?

Reply:
I'm just suggesting an entirely excessive level of markdown formatting. I find the pasted TSV a little difficult to read because of the varying indentation, so I was thinking you could have a nicely formatted markdown table _as well as_ the raw TSV that you currently have. So as not to clutter the page too much, the latter could be hidden in an expandable dropdown.

Like I say, this is totally excessive, so feel free to ignore. It was only if you wanted to geek out on some markdown 🤓 😅 

It didn't help that I couldn't paste an example, as the triple-backticks you have for the code block messed up the triple-backticks for my written example.

Reply:
I can write the markdown in a comment though, rendered by GitHub for a preview of what I mean:

---

| group   | condition | control | reads |
|---------|-----------|---------|-------|
| MOLM-13 | DMSO   | 1 | MOLM-13_dmso_1.fq.gz   |
| MOLM-13 | DMSO   | 1 | MOLM-13_dmso_2.fq.gz   |
| MOLM-13 | DMSO   | 1 | MOLM-13_dmso_3.fq.gz   |
| MOLM-13 | NVP_hi | 0 | MOLM-13_nvp.hi_1.fq.gz |
| MOLM-13 | NVP_hi | 0 | MOLM-13_nvp.hi_2.fq.gz |
| MOLM-13 | NVP_hi | 0 | MOLM-13_nvp.hi_3.fq.gz |
| MOLM-13 | NVP_lo | 0 | MOLM-13_nvp.lo_1.fq.gz |
| MOLM-13 | NVP_lo | 0 | MOLM-13_nvp.lo_2.fq.gz |
| MOLM-13 | NVP_lo | 0 | MOLM-13_nvp.lo_3.fq.gz |
| OCIAML3 | DMSO   | 1 | OCIAML3_dmso_1.fq.gz   |
| OCIAML3 | DMSO   | 1 | OCIAML3_dmso_2.fq.gz   |
| OCIAML3 | DMSO   | 1 | OCIAML3_dmso_3.fq.gz   |
| OCIAML3 | NVP_hi | 0 | OCIAML3_nvp.hi_1.fq.gz |
| OCIAML3 | NVP_hi | 0 | OCIAML3_nvp.hi_2.fq.gz |
| OCIAML3 | NVP_hi | 0 | OCIAML3_nvp.hi_3.fq.gz |
| OCIAML3 | NVP_lo | 0 | OCIAML3_nvp.lo_1.fq.gz |
| OCIAML3 | NVP_lo | 0 | OCIAML3_nvp.lo_2.fq.gz |
| OCIAML3 | NVP_lo | 0 | OCIAML3_nvp.lo_3.fq.gz |

<details>
<summary>Raw TSV:</summary>

```bash
group  condition control reads
MOLM-13 DMSO  1 MOLM-13_dmso_1.fq.gz
MOLM-13 DMSO  1 MOLM-13_dmso_2.fq.gz
MOLM-13 DMSO  1 MOLM-13_dmso_3.fq.gz
MOLM-13 NVP_hi  0 MOLM-13_nvp.hi_1.fq.gz
MOLM-13 NVP_hi  0 MOLM-13_nvp.hi_2.fq.gz
MOLM-13 NVP_hi  0 MOLM-13_nvp.hi_3.fq.gz
MOLM-13 NVP_lo  0 MOLM-13_nvp.lo_1.fq.gz
MOLM-13 NVP_lo  0 MOLM-13_nvp.lo_2.fq.gz
MOLM-13 NVP_lo  0 MOLM-13_nvp.lo_3.fq.gz
OCIAML3 DMSO  1 OCIAML3_dmso_1.fq.gz
OCIAML3 DMSO  1 OCIAML3_dmso_2.fq.gz
OCIAML3 DMSO  1 OCIAML3_dmso_3.fq.gz
OCIAML3 NVP_hi  0 OCIAML3_nvp.hi_1.fq.gz
OCIAML3 NVP_hi  0 OCIAML3_nvp.hi_2.fq.gz
OCIAML3 NVP_hi  0 OCIAML3_nvp.hi_3.fq.gz
OCIAML3 NVP_lo  0 OCIAML3_nvp.lo_1.fq.gz
OCIAML3 NVP_lo  0 OCIAML3_nvp.lo_2.fq.gz
OCIAML3 NVP_lo  0 OCIAML3_nvp.lo_3.fq.gz
```

</details>

Reply:
Looks indeed better ;)